### DISCUSSION TITLE
Connection status?

### DISCUSSION THREAD

**Discussion by KristijanZic** [serverpod/serverpod#2716] [Q&A] [answered]
Is there a build in way to watch connection status with Serverpod? I want to be able to show the connection with the server status icon to the user in the flutter app. In case a server is down I want to be able to react to it. Is there a built in way to do that?

**Comment by donywan** [comment]
Do you want to monitor whether the serverpod server service is down?

**Reply by KristijanZic** [reply]
Yes! :) If it's down or for some reason the communication from the client can't be established with the serverpod service. In which case I'd probably prompt the user to contact his administrator or somebody in charge.

**Reply by donywan** [reply, marked as answer]
You can set `connectionTimeout` and `onFailedCall` when create `Client` instance.
